part-iv-advanced-tuning/advanced-customizations.md

```diff
@@ -154,6 +153,12 @@ public class MethodCommandHandlerDefinition implements HandlerEnhancerDefinition
 
 To skip all handling of the handler then just throw an exception.
 
+Since 3.3 Axon version it is possible to configure `HandlerDefinition` with Axon `Configuration`. If you are using Spring Boot defining `HandlerDefintion`s and `HandlerEnhancerDefinition`s as beans is sufficient (Axon autoconfiguration will pick them up and configure within Axon `Configuration`).
+
+```java
```

Comment:
For all the other configuration code samples, we also provide the `Spring Boot Auto Configure` situation.
I know that's just showing a bean creation method, but nonetheless I think it is fair to be consistent on this.